Keith Ho/week_4/day_1/myREPL/keith_repl.rb

```diff
@@ -0,0 +1,12 @@
+
+ruby -r coderay -r readline -e "
+bnd = binding()
+  while input = Readline.readline("keith~>>", true)
+    begin
```

Comment:
This whole block can be edited to: 
```begin
result = bnd.eval input
puts CodeRay.encode(result.inspect, :ruby, :terminal) // this is getting user input which is at risk of error so you wrap it in a begin/rescue block
    rescue StandardError => e  //this handles and displays the error message
      puts "\e[31m#{e.class}:\e[0m #{e.message}"
      //else statement is only required if all goes well, it will be where the program branches to
end
 ```

Reply:
Thanks so much for your help! Rowena